wgpu-types/src/lib.rs

```diff
@@ -1540,6 +1553,8 @@ impl TextureViewDimension {
 ///
 /// Corresponds to [WebGPU `GPUBlendFactor`](
 /// https://gpuweb.github.io/gpuweb/#enumdef-gpublendfactor).
+/// For the extended values see the OpenGL extension
+/// [ARB blend func extended][https://registry.khronos.org/OpenGL/extensions/ARB/ARB_blend_func_extended.txt]
```

Comment:
```suggestion
```
Since we'll document this on the feature flag which we reference on the enum variants we probably don't need the reference here.